README.md

```diff
@@ -3,47 +3,79 @@
 
 The Epsilon Dashboard displays information on the screens.
 
-## Switching Modes
+## Project Setup
 
-There are three different modes for the dashboard: Display mode, Debug Display mode and race mode. The default mode is display mode.
-To run the application in different modes, navigate to the directory where you made the executable file for the dashboard. 
+In this repo, there are dependencies needed for before you will be able to build or run anything.
 
-To run the application in display mode, run the command:
-	`./EpsilonDashboard`
+To install these dependencies, run the setup script:
 
-To run the application in debug display mode, run the command:
-	`./EpsilonDashboard -d`
-  
-To run the application in race mode, add the -r flag at the end:
-  `./EpsilonDashboard -r`
+`sudo ./EpsilonDashboardSetup.sh`
 
-## Setting up Rabbitmq
+This will install the [RabbitMQ](https://www.rabbitmq.com/) server and the [Conan](https://conan.io/) package manager. See the links for more information.
 
-In this repo, there are dependencies needed for before you will be able to build or run anything.
+## Building
 
-To install these dependencies, run the command:
-	`./EpsilonDashboardSetup.sh`
+### QT Creator
 
-This will install the [RabbitMQ](https://www.rabbitmq.com/) server and the [Conan](https://conan.io/) package manager. See the links for more information.
+When first setting up your project with QT creator, you must first add a custom step to allow Conan to install dependencies.
+
+1. Navigate to `Projects -> Build`
+2. In `Build Steps`, select `Add Build Step -> Custom Process Step`
```

Comment:
Is there any way to package this process into the GitHub repo?

Reply:
I think you can put it in a `.pro.shared` file, but if the user has even a slightly different build settings it'll just get thrown out?
https://stackoverflow.com/questions/13994495/how-to-specify-build-deployment-and-run-configuration-in-a-qt-creator-pro-sh